app/repositories/SessionRepository.scala

```diff
@@ -31,7 +31,7 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 
-
+@Singleton
```

Comment:
Were creating the binding as Singleton so it's essentially the same https://github.com/hmrc/emcs-tfe-create-movement-frontend/blob/main/app/config/Module.scala#L36

Reply:
True :)

Reply:
https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/110#issuecomment-1782605555 looks like we need it to make the platops bot happy